index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+<h1>Hello World!</h1>
+  <header id="header">
```

Comment:
Job well done so far, 👏 , however, Please ensure that the <h1> tag is placed inside <header> tag, wrapping the <h1> tag within a <header> tag not only helps create a more organized and meaningful structure for your HTML document but also improves accessibility, SEO, and maintainability of your website or web application. 🙂